boards/nxp/rd_rw612_bga/CMakeLists.txt

```diff
@@ -14,4 +14,5 @@ if(CONFIG_NXP_RW6XX_BOOT_HEADER)
   zephyr_compile_definitions(BOARD_FLASH_SIZE=CONFIG_FLASH_SIZE*1024)
   zephyr_library()
   zephyr_library_sources(MX25U51245GZ4I00_FCB.c)
+  zephyr_library_sources(init.c)
```

Comment:
This should be moved outside the CONFIG as we need this file even when we have a bootloader.

Reply:
Thank you again for pointing this out, updated.

Reply:
Thank you again for pointing this out, updated.